### PR TITLE
Add BoolParam class

### PR DESCRIPTION
**Pull request by tomislater**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,11 @@
 
 
 # graceful
-`graceful` is documentation centered falcon REST toolkit. It is highly inspired
```
Comment (tomislater) [inline README.md]: Uh, I have created the branch based on master.

Comment (swistakm) [inline README.md]: No problem at all :)

**Comment by swistakm** [comment]
thanks for PR!

**Comment by swistakm** [comment]
I have made small modification after the merge to include also explicit list of True values to avoid user's confusion in case of typos. See: 613a16627

**Comment by tomislater** [comment]
+1

**Pull request merged**